mlir/test/Dialect/LLVMIR/alias.mlir

```diff
@@ -0,0 +1,127 @@
+// RUN: mlir-opt %s -split-input-file | FileCheck %s
```

Comment:
```suggestion
// RUN: mlir-opt %s -split-input-file --verify-roundtrip | FileCheck %s
```
Let's add the verify round trip here to ensure we do a full round trip not just parsing and printing. This increases chances to unveil parsing order problems like the possible swap between unnamed_addr and thread_local, if there is a test case.

Reply:
done